source/index.md.erb

```diff
@@ -72,7 +72,7 @@ API access is available on Pro, Enterprise and Business plans of 10000ft. As of
 
 # Need help?
 
-* Contact us via our [Support Page](https://help.smartsheet.com/contact/10000ft)
+* Contact us via our [Support Page](https://help.smartsheet.com/contact/resource-management)
```

Comment:
Same with below question and others across the changes, related to support page.